jekyll/_cci2/single-box.md

```diff
@@ -0,0 +1,138 @@
+  <option value="us-east-1" selected="selected">us-east-1</option>
+  <option value="us-east-2">us-east-2</option>
+  <option value="us-west-1">us-west-1</option>
+  <option value="us-west-2">us-west-2</option>
+  </select>
+  <a id="ami-go" href="" class="btn btn-success" data-analytics-action="{{ site.analytics.events.go_button_clicked }}" target="_blank">Go!</a>
+
+<script>amiUpdateSelect();</script>
+</li>
+
+<li>During the Configure Security Group step, open port 22 for SSH, 80 for HTTP, 443 for HTTPS, and 8800 for Custom TCP.
+</li>
+
+<li>(Optional) To enable developers to SSH into builds for debugging purposes, open ports 64535-65535 for Custom TCP.
+</li>
+
+<li>After the VM is lauched, go to the public or private IP address or hostname for the VM and click Get Started to complete the rest of the guided installation process.
+</li> 
+
+<li>Choose an SSL certificate option. By default, all machines in a CircleCI installation verify SSL certificates for the GitHub Enterprise instance. If you're using a self-signed cert,
+or using a custom CA root, select the HTTPS (with self-signed certificate) option in the System Console at port 8800.
+You also need to export <code>CIRCLE_IGNORE_CERT_HOST=insecure-ghe.example.com</code> on builder machines replacing <code>insecure-ghe.example.com</code> with the host of your GitHub Enterprise instance. See [this doc]({{site.baseurl}}/enterprise/docker-builder-config/) for details on setting builder machine environment variables.
+</li>
+
+<li>
+Upload the license file and set the admin password.
+</li>
+<li>
+Register CircleCI as a new OAuth application in GitHub.com at <a href="https://github.com/settings/applications/new">https://github.com/settings/applications/new</a> or in the GitHub Enterprise Settings using the IP address of the AWS instance from Step 4 for the Homepage URL and using <code>http(s)://AWS instance IP address/auth/github</code> as the Authorization callback URL. Click the Register Application button.
+</li>
+<li>
+Copy the Client ID from GitHub and paste it into the entry field for GitHub Application Client ID.
+</li>
+<li>
+Copy the Secret from GitHub and paste it into the entry field for GitHub Application Client Secret and click Test Authentication.
+</li>
+<li>
+Agree to the license agreement and save. The application start up process begins by downloading the ~160 MB docker image, so it may take some time to complete. 
+</li>
+
+<li>Open the CircleCI app and click Get Started to authorize your GitHub account. The Add Projects page appears where you can select a project for your first build. 
+</li>
+</ol>
+
+### AWS Installation Video Tutorial
+
+<p style="font-size: 20px; text-align: center">Following is a short video walkthrough of the entire install process on AWS:</p>
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/m4plGZmZkj4" frameborder="0" allowfullscreen style="display: block; margin: 20px auto;"></iframe>
+
+
+## Installation in a Data Center
```

Comment:
We should hide this for 2.0 launch, as we don't yet support non-AWS on the 2.0 server

Reply:
this card https://circleci.atlassian.net/browse/CIRCLE-4235 makes me think otherwise, so I will wait for someone else to confirm

Reply:
That's only one part of the non-AWS story. As of launch we won't have non-AWS support. It's coming soon, though.

Reply:
ok, removed